helmexec/runner.go

```diff
@@ -0,0 +1,30 @@
+package helmexec
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+)
+
+const (
+	tmpPrefix = "helmfile-"
+	tmpSuffix = "-exec"
+)
+
+type Runner interface {
```

Comment:
Can we name `CliRunner` as `shellExecer` and omit this interface? The rationale is that, at this point, we don't have any concrete benefit by factoring out an interface.

Reply:
I tried to do this, but ultimately failed to see how I can use a mocked runner without an interface. We need to set the runner explicitly in `helmexec.New()`, so it needs to be an interface. I renamed it to `ShellRunner` though, to honour your sentiment :)

Maybe I need to go back to the books about Go, I feel I am not seeing these things clearly.

Reply:
Good point! I had definitely overlooked that. Thx for the rename, too 😄